app/views/plates/creator/_printed_barcodes.html.erb

```diff
@@ -0,0 +1,20 @@
+<table>
+  <thead>
+    <tr>
+      <th><%= @creator.created_plates.first[:source].purpose.name %></th>
+      <% @creator.created_plates.first[:destinations].each do |dest| %>
+        <th><%= dest.purpose.name %></th>
+      <% end %>
+    </tr>
+  </thead>
+  <tbody>
+    <% @creator.created_plates.each do |entry| %>
+      <tr>
+        <td><%= entry[:source].barcodes.first.barcode %> </td>
+        <% entry[:destinations].each do |destination| %>
+            <td><%= destination.barcodes.first.barcode %>  </td>
+        <% end %>
+      </tr>
+    <% end %>
+  </tbody>
+</table>
```

Comment:
Why is 'destinations' plural? How do you create multiple child plates from one source plate using this page?

Reply:
For some combinations of source plate purposes and destination purposes the creators can create more than one plate. For example, the added screenshot the source is the working dilution WD... barcode and the destination plate purposes is Pico Assay Plates selection

Reply:
Thanks. From looking at the table structure, it looks like all your source plates / racks have to be of the same 'purpose' - is that correct?

Reply:
Good catch! yes, that could be a problem if we  print from TR48 along with TR96...
Destinations will be fine, but Sources may have different plate purpose.

Reply:
So I'll change the table structure to show the plate purpose of every source

Reply:
Yeah, could just call the first column 'Source' or something generic